docs/tokenomics/token-holders/nns-app-quickstart.md

```diff
@@ -264,6 +264,50 @@ To disburse a neuron and return its ICP utility tokens:
 
 - #### Step 4:  Verify the transaction information, then click **Confirm and Send**. 
 
+## Neuron following
+
+To maximize neuron rewards and fully utilize the NNS's liquid democracy, neuron owners are encouraged to vote on every proposal submitted to the NNS. However, it may be hard for neuron owners to manually vote on every single proposal submitted, since several proposals may be submitted at one time, or neuron owners may not have the necessary level of technical understanding to evaluate proposals themselves. To aid in this, the NNS supports **neuron following**.
+
+A neuron can be configured to vote automatically by following the voting decision made by a group of other neurons. For example, if the neuron follows a group that votes to adopt a proposal with more than half of the followees voting in favorite of adoption, the neuron following that group will vote to approve the proposal as well. If the neuron follows a group that votes to reject a proposal with more than half of the followees voting to reject, the neuron following the group will vote to reject the proposal. 
```

Comment:
Should we add a more explicit remark wrt the following:
I think many users thing following more neurons is better as it will ensure that a vote is cast for them. In many cases this is not true however: If a neuron follows 10 followees on a topic it will _only_ vote if 6 out of these 10 vote the same way. This means, that if 5 vote, then the following neuron will not cast a vote.
So if there are some reliable neurons, it can be the better option to actually follow less neurons.
Of course there can be other motivations to follow multiple experts - but I wonder if the above is something that some users don't understand well. Maybe we can also just not make any recommendations but mention an example e.g., with the 10 followees?